projects/akita-ng-fire/src/lib/utils/sync-from-action.ts

```diff
@@ -29,8 +29,7 @@ export function removeStoreEntity(storeName: string, entityIds: string | string[
 /** Update one or several entities in the store */
 export function updateStoreEntity(storeName: string, entityIds: string | string[], data: any) {
   removeStoreEntity(storeName, entityIds);
-  runEntityStoreAction(storeName, EntityStoreAction.AddEntities, add => add(data))
-  runEntityStoreAction(storeName, EntityStoreAction.UpdateEntities, update => update(entityIds, data))
+  runEntityStoreAction(storeName, EntityStoreAction.AddEntities, add => add(data));
```

Comment:
You might want to run both in a transaction to avoid jump in the page.

Reply:
Also you if `entityIds` is  `["1", "2"]` & data `{ text: "Hello" }` using `add(data)` will not work as expected.
Check if it's possible but I think you want to do : 
```typescript
add => add(entityIds, data)
```